src/Wrapper.cpp

```diff
@@ -21,13 +22,18 @@ ZIVID_PYTHON_MODULE // NOLINT
     module.attr("__version__") = pybind11::str(ZIVID_PYTHON_VERSION);
 
     ZIVID_PYTHON_WRAP_DATA_MODEL(module, Settings);
+    ZIVID_PYTHON_WRAP_DATA_MODEL(module, Settings2D);
     ZIVID_PYTHON_WRAP_DATA_MODEL(module, CameraState);
     ZIVID_PYTHON_WRAP_DATA_MODEL(module, FrameInfo);
     ZIVID_PYTHON_WRAP_DATA_MODEL(module, CameraIntrinsics);
 
     ZIVID_PYTHON_WRAP_CLASS_AS_SINGLETON(module, Application);
     ZIVID_PYTHON_WRAP_CLASS_AS_RELEASABLE(module, Camera);
     ZIVID_PYTHON_WRAP_CLASS_AS_RELEASABLE(module, Frame);
+    ZIVID_PYTHON_WRAP_CLASS_AS_RELEASABLE(module, Frame2D);
+
+    ZIVID_PYTHON_WRAP_CLASS_BUFFER_AS_RELEASABLE(module, Image);
```

Comment:
Move down so we are in line with the original system of grouping the same macros together.